yb-voyager/cmd/common.go

```diff
@@ -225,14 +234,21 @@ func displayExportedRowCountSnapshot(snapshotViaDebezium bool) {
 					schema = tableParts[0]
 					table = tableParts[1]
 				}
-				uitable.AddRow(schema, table, exportedRowCount[key])
+				displayTableName := table
+				if source.DBType == POSTGRESQL && leafPartitions[key] != nil && msr.IsExportTableListSet {
```

Comment:
What is the purpose of `msr.IsExportTableListSet`

Reply:
This is to handle the second case if a subset of the table-list is passed, So I am just doing a simple thing to print the leaf partitions in case the table list is passed to export nothing extra for the subset and all. I think if a user has passed a table-list (event the root table is passed) then we should tell what all will be exported or imported.